packages/carbon-web-components/src/components/structured-list/structured-list.scss

```diff
@@ -31,6 +31,10 @@ $css--plex: true !default;
   ::slotted(#{$prefix}-structured-list-header-cell-skeleton) {
     @extend .#{$prefix}--structured-list-th;
   }
+
+  ::slotted(#{$prefix}-structured-list-header-cell) {
+    padding-left: 1rem;
```

Comment:
Let's use spacing tokens for all instances of these units?

Reply:
updated and fixed some new spacing issues I found

Reply:
@emyarod you may have just fixed some of the spacing things I caught and just posed in my review 😄